### DISCUSSION TITLE
Morphos

### DISCUSSION THREAD

**Discussion by maxbengtzen** [selfhst/icons#79] [New Icon Request]
Repo: https://github.com/danvergara/morphos
Link to logo in repo: https://github.com/danvergara/morphos/blob/main/screenshots/morphos.jpg

**Comment by shollyethan** [comment]
Reached out to the dev for a higher quality format than JPG. Will comment here when I receive something!

**Comment by shollyethan** [comment]
Morphos has been added - thanks!
